PathManager/Src/PM_States.cpp

```diff
@@ -0,0 +1,300 @@
+#include "PM_States.hpp"
+
+/***********************************************************************************************************************
+ * Static Member Variable Declarations
+ **********************************************************************************************************************/
+
+namespace PM {
+SM_PM_Commands CommsWithSystemManager::incomingData;
+//SFOutput_t SensorFusion::sfOutputData;
+//IMU_Data_t SensorFusion::imudata;
+AM::AttitudeManagerInput CommsWithAttitude::receivedData;
+
+
+// TAKEOFF STAGE VARIABLES 
+WaypointData * TakeoffStage::currentLocation;    
+WaypointData * TakeoffStage::targetWaypoint;
+AM::AttitudeManagerInput TakeoffStage::takeoffDataForAM; 
+
+// LANDING STAGE VARIABLES 
+WaypointData * LandingStage::currentLocation;
+WaypointData * LandingStage::targetWaypoint;
+AM::AttitudeManagerInput LandingStage::landingDataForAM;
+
+
+constexpr int LANDING_TIME_THRESHOLD {5};
+
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+void CommsWithSystemManager::execute(PathManager* pathMgr)
+{
+    //GetSMIncomingData();
+    /*
+
+    */
+
+    
+}
```

Comment:
Is there something missing here? Since the thread that calls PM handles the PM side of the message queues is this necessary?